README.md

```diff
@@ -57,4 +57,4 @@ This is not an official Google product.
 This is an experimental project - expect plenty of rough edges and bugs, and 
 no support.
 
-For discussions, there is a Slack channel (#shenzhen-go) at gophers.slack.com.
+For discussions, there is a Slack channel (#shenzhen-go) at [gophers.slack.com](https://gophers.slack.com/).
```

Comment:
If you’re going to link to Gophers Slack, you could consider linking directly to the #shenzhen-go channel instead:

https://gophers.slack.com/messages/shenzhen-go

Reply:
+1, perhaps it should just be:

For discussions, there is [a Slack channel](https://gophers.slack.com/messages/shenzhen-go).